external/source/exploits/CVE-2022-46689/vm_unaligned_copy_switch_race.c

```diff
@@ -0,0 +1,359 @@
+// from https://github.com/apple-oss-distributions/xnu/blob/xnu-8792.61.2/tests/vm/vm_unaligned_copy_switch_race.c
```

Comment:
Just wanted to say this PR is awesome, thanks so much for the contribution! Glad to see this exploit is open source :)

Reply:
Hey @cdelafuente-r7 I've added the related license to the LICENSE file. This is my first addition to the LICENSE file so please let me know if you think I've done it correctly or if any improvements could be made 🙏